tests/test_main.py

```diff
@@ -131,6 +137,20 @@ def test_m2m_clear(self, model, m2m_model):
         data = json.loads(crud_event.object_json_repr)[0]
         assert [str(d) for d in data["fields"]["test_m2m"]] == []
 
+    def test_custom_manager(self, custom_manager_model):
+        obj = custom_manager_model.objects.create(deleted=True)
+        crud_event_qs = CRUDEvent.objects.filter(
+            object_id=obj.id, content_type=ContentType.objects.get_for_model(obj)
+        )
+        assert crud_event_qs.count() == 1
+
+        obj.name = "this should not fail"
```

Comment:
I'm currently having a bit of trouble understanding how this test is proving the fix. Can you add more information in the test or perhaps outline more of it in discussion?